linkcheck/HtmlParser/Makefile

```diff
@@ -2,7 +2,7 @@
 # reentrant bison parser support and uses features of bison >= 3.0.x
 LEX = flex
 YACC = bison
-PYINCLUDE=-I/usr/include/python2.7
+PYINCLUDE=-I/usr/include/python
```

Comment:
FWIW my machine (Ubuntu 18.10) does not have a /usr/include/python.  It has /usr/include/python2.7 and /usr/include/python3.* for various values of *.

I wonder if we should maybe use something like `$(pkg-config --cflags python)` (which on my machine prints -I/usr/include/python2.7 -I/usr/include/x86_64-linux-gnu/python2.7)?

(Or we could merge that PR that replaces linkcheck/HtmlParser with BeautifulSoup.)

Reply:
I don't have /usr/include/python either, and nor does `pkg-config --cflags python` work. Supporting both Python 2 and 3 is complicated, and varies between distrubutions...

On the other hand, what does this file need to do? Supply a clean instruction? Run the awk?
Because setup.py does seem to do the actual building using Extension - which is the bit that needs the include.